pilot/prompts/components/files_list.prompt

```diff
@@ -1,6 +1,6 @@
 {% if files|length > 0 %}Here are files that are currently implemented:
 ---START_OF_FILES---{% for file in files %}
-**{{ file.path }}/{{ file.name }}**:
+**{{ file.path }}/{{ file.name }}** ({{ file.lines_of_code }} lines of code):
```

Comment:
You also need to update this for telemetry not to break:

https://github.com/Pythagora-io/gpt-pilot/blob/8327d05ab3a424002c5d043134e5b340a3e81879/pilot/utils/llm_connection.py#L324